### PR TITLE
EA-76: Ignore DateMapperTest and quit depending on distro.

### PR DESCRIPTION
**Pull request by mks-d**
This test just fails with Java 7, see
https://issues.openmrs.org/browse/EA-76?focusedCommentId=236199&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-236199
  modified:   api/src/test/java/org/openmrs/module/emrapi/encounter/DateMapperTest.java
No more depency on distro 2.3.
  modified:   omod/pom.xml
  modified:   pom.xml

### CODE REVIEW

**Comment by mks-d** [comment]
@dkayiwa , as agreed, could you release version 1.12.1 out of this commit?

File: api/src/test/java/org/openmrs/module/emrapi/encounter/DateMapperTest.java [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
-import static org.junit.Assert.assertNotNull;
+import org.junit.Ignore;
+import org.junit.Test;
 
+/*
+ * mks-d:
+ * This test won't run on Java 7, see the link below.
+ * https://issues.openmrs.org/browse/EA-76?focusedCommentId=236199&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-236199
+ */
+@Ignore
```
Comment (dkayiwa) [inline api/src/test/java/org/openmrs/module/emrapi/encounter/DateMapperTest.java]: I thought you would put this text right in the ignore annotation.

**Comment by mks-d** [comment]
@dkayiwa , I didn't know that `@Ignore` could take a parameter...
Done now, could you have look?

**Comment by dkayiwa** [comment]
Ok, done.

**Comment by mks-d** [comment]
Thanks @dkayiwa .
Last thing... could you release EMR API 1.12.1 then? :blush:

**Comment by dkayiwa** [comment]
@mks-d how do you tell whether it is released or not?  :)

**Comment by mks-d** [comment]
Ah! Thanks a million.

@rbuisson , Daniel has released EMR API 1.12.1, we can now depend on that one where needed.

**Pull request merged**